fenix/app/src/main/res/values/dimens.xml

```diff
@@ -11,6 +11,10 @@
     <dimen name="mozac_browser_menu_width_max" tools:ignore="UnusedResources">314dp</dimen>
     <dimen name="mozac_browser_menu_corner_radius">8dp</dimen>
     <dimen name="mozac_browser_menu2_corner_radius">8dp</dimen>
+    <dimen name="mozac_browser_menu_item_checkbox_container_padding_end" tools:ignore="UnusedResources">0dp</dimen>
```

Comment:
Why do we need to add these?

Reply:
It's kinda the same [question](https://github.com/mozilla-mobile/firefox-android/pull/2697#discussion_r1252732024).

In short, the container has end spacing, and the checkbox button has end spacing, so the text doesn't get enough space. I extended styling on A-C level, while keeping the existing design, so we could override things we want to look different on the fenix level. 

Basically, I am removing end spacing for the container for the element that has a text and a checkbox button.

Reply:
No I mean why is it `tools:ignore="UnusedResources"`?